pkg/secret/secret.go

```diff
@@ -33,6 +33,12 @@ func NewBuilder(apiClient *clients.Settings, name, nsname string, secretType cor
 		"Initializing new secret structure with the following params: %s, %s, %s",
```

Comment:
Please move secret updates to a separate PR

Reply:
yea,it had to be a separate mr, it's by mistake here. thanks!